manifests/charts/gateways/istio-egress/templates/deployment.yaml

```diff
@@ -50,11 +50,18 @@ spec:
         prometheus.io/path: "/stats/prometheus"
         {{- end }}
         sidecar.istio.io/inject: "false"
+        {{- if .Values.gateways.sgx.enabled }}
+        sgx.intel.com/enabled: "true"
+        sgx.intel.com/quote-provider: "aesmd"
+        {{- end }}
+        {{- if .Values.gateways.sgx.certExtensionValidationEnabled }}
+        sgx.intel.com/cert-extension-validation-enabled: "true"
+        {{- end }}
 {{- if $gateway.podAnnotations }}
 {{ toYaml $gateway.podAnnotations | indent 8 }}
 {{ end }}
     spec:
-{{- if not $gateway.runAsRoot }}
+{{- if not (or $gateway.runAsRoot .Values.gateways.sgx.enabled) }}
```

Comment:
Do I read this correctly that you need root access? Accessing devices from non-root containers should be have been fixed in both containerd and cri-o for a while now.

Reply:
Yes, we shouldn't need root access, I removed root in ingress/egress and injection template deployment files, thanks for the correction.